unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldBridge/BuilderInWorldProtocol.cs

```diff
@@ -0,0 +1,132 @@
+using DCL.Models;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BuilderInWorldProtocol 
+{
+    #region Class Declarations
+
+    [System.Serializable]
+    public class QuaternionRepresentantion
```

Comment:
Please bring more attention to naming typos, here is another one.

```suggestion
    public class QuaternionRepresentation
```